EnglishTranslation-master/QUEST_JOBSTEP.tsv

```diff
@@ -3070,13 +3070,13 @@ QUEST_JOBSTEP_20150730_003069	This was when the Coral Plain was still sea.{nl}I
 QUEST_JOBSTEP_20150730_003070	I won at that time, but I fell his necklace which I obtained after beating him into the sea.{nl}The sea was very deep at then, but I may able to find it now.
 QUEST_JOBSTEP_20150730_003071	If I could find that, the people who said my words were bluffs will be surprised.{nl}It will be around 32_1 of the Coral Plain. You are the Corsair who I acknowledged so you will find it easily.  
 QUEST_JOBSTEP_20150730_003072	Good!{nl}I will teach you everything. 
-QUEST_JOBSTEP_20150730_003073	Well done. It could have been better if we were to find it a little sooner.{nl}I will teach you the new formation as a reward. It's precise so it's much stronger.
-QUEST_JOBSTEP_20150730_003074	If you are a Doppelsoeldner, you should do something for the kingdom.{nl}I want to do it myself, but since it is being done quietly, I will be spotted if I do it myself.
-QUEST_JOBSTEP_20150730_003075	Thanks for saying so. Please find the envoy of Ramin General secretly.{nl}His last contact was from 36_2 of Pilgrim's way.
-QUEST_JOBSTEP_20150730_003076	Sad. I guess he was attacked by the monsters.{nl}Anyway, since you've become a member of Doppensoeldners, I will support you with my best.
-QUEST_JOBSTEP_20150730_003077	Ah, and please keep this secret to other people.{nl}I gotta find an other way to contact Ramin general.
+QUEST_JOBSTEP_20150730_003073	Well done. It could have been better if we were able to find that a little sooner.{nl}I will teach you the new formation as a reward. It's precise, so it's much stronger.
+QUEST_JOBSTEP_20150730_003074	If you are a Doppelsoeldner, you should do something for the kingdom.{nl}I want to do it myself, but since it has to be done quietly, I don't trust my ability to take care of the situation.
+QUEST_JOBSTEP_20150730_003075	Thanks for saying so. Please find the envoy of Ramin General as secretly as possible.{nl}His last contact was from 36_2 of Pilgrim's way.
+QUEST_JOBSTEP_20150730_003076	Sad. I guess he was attacked by the monsters.{nl}Anyway, since you've become a member of Doppelsoeldners, I will support you with my all.
+QUEST_JOBSTEP_20150730_003077	Ah, and please keep this a secret.{nl}I gotta find an other way to contact Ramin General.
 QUEST_JOBSTEP_20150730_003078	If you've explored the area completely in the past, just record it in the orb.{nl}After exploring all the areas in Crystal Mine, use the orb to record your past.
-QUEST_JOBSTEP_20150730_003079	Explore all the regions in the 1st and 2nd districts with this orb.{nl}If you've explored the areas completely in the past, just record it in the orb.{nl}After exploring all the area, use the orb to record your past.
+QUEST_JOBSTEP_20150730_003079	Explore all the regions in the 1st and 2nd districts with this orb.{nl}If you've completely explored the areas already, just record it in the orb.{nl}After obtaining a 100% exploration rate for these areas, use the orb to record your past.
```

Comment:
Explore all the regions in the 1st and 2nd districts with this orb -> Explore all the areas in District 1 and 2 with this orb.